chess_library.py

```diff
@@ -1,18 +1,16 @@
-from code import chess_move
-from code import chess_position
+from chess_move import ChessMove
+from chess_position import ChessPosition
 
-class ChessLibrary:
```

Comment:
Add this line back, we should still be writing methods for the `ChessLibrary` class.